e2e/cypress/tests/integration/enterprise/ldap_group/invite_bot_spec.ts

```diff
@@ -43,7 +49,12 @@ describe('Group Synced Team - Bot invitation flow', () => {
 
         // # Logout sysadmin and login as an LDAP Group synced user
         cy.apiLogout();
-        cy.apiLogin({username: 'test.one', password: 'Password1'});
+
+        const user = {
+            username: 'test.one',
+            password: 'Password1',
+        } as UserProfile;
```

Comment:
I'd suggest to use [the one we have under `Cypress` namespace](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/index.d.ts#L31) so we import only in one place.
```suggestion
        } as Cypress.UserProfile;
```

Reply:
Done 👍